templates/redmine/0/rancher-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '2'
+catalog:
+  name: Redmine
+  version: 0.1.2
+  description: |
+    Redmine
+  questions:
+  - variable: "PORT"
+    description: "external port to access redmine on"
+    label: "Port"
+    required: true
+    default: "11080"
+    type: "int"
+  - variable: "TZ"
+    description: "Timezone to use for the server"
+    label: "Timezone"
+    required: true
+    default: "America/Chicago"
+    type: "string"
+  - variable: "DB_PASS"
+    description: "Database Password to use for the server"
+    label: "DB Password"
+    required: true
+    default: "password123"
+    type: string
+  - variable: "REDMINE_SECRET_TOKEN"
+    description: "REDMINE_SECRET_TOKEN"
+    label: "REDMINE_SECRET_TOKEN"
+    required: true
+    default: "REALLYlongrandomhashkjekkckksixci"
+    type: "string"
+
+services:
+  redmine:
+    scale: 1
+    retain_ip: true
```

Comment:
If you use traefik, it's just fine. The point is avoid the expose of ports directly from the service. Could you please add a README.md explaining that the package needs a traefik just for accessing??